### PR TITLE
[SMALLFIX] Port JournalShutdownIntegrationTest

### PR DESCRIPTION
**Pull request by uronce-cc**

### CODE REVIEW

File: integration-tests/src/test/java/tachyon/master/next/JournalShutdownIntegrationTest.java [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+    System.clearProperty("fs.hdfs.impl.disable.cache");
+  }
+
+  @Before
+  public final void before() throws Exception {
+    System.setProperty("fs.hdfs.impl.disable.cache", "true");
+  }
+
+  private FileSystemMaster createFsMasterFromJournal() throws IOException {
+    // TODO
```
Comment (uronce-cc) [inline integration-tests/src/test/java/tachyon/master/next/JournalShutdownIntegrationTest.java]: This method should be the same with [this method in another PR](https://github.com/amplab/tachyon/pull/1443/files#diff-5deda5f174aef4333820ae72681496afR84), would update after that PR is merged.

Comment (gpang) [inline integration-tests/src/test/java/tachyon/master/next/JournalShutdownIntegrationTest.java]: Does this have to be updated now?

**Comment by gpang** [comment]
I just had a minor comment. Thanks @uronce-cc !

**Comment by uronce-cc** [comment]
@gpang PTAL

**Comment by gpang** [comment]
LGTM

**Pull request merged**